include/prism/node.h

```diff
@@ -123,6 +130,7 @@ PRISM_EXPORTED_FUNCTION void pm_visit_node(const pm_node_t *node, bool (*visitor
  * @param node The node to visit the children of.
  * @param visitor The callback to call for each child node.
  * @param data An opaque pointer that is passed to the visitor callback.
+ * \public \memberof pm_node
```

Comment:
Same here, this is more related to the visitor, so I don't think this should be associated with a particular node.